packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -699,6 +699,24 @@ export default class Meetings extends WebexPlugin {
     }
   }
 
+  /**
+   * API to toggle TCP reachability, needs to be called before webex.meetings.register()
+   * @param {Boolean} newValue
+   * @private
+   * @memberof Meetings
+   * @returns {undefined}
+   */
+  private _toggleTcpReachability(newValue: boolean) {
+    if (typeof newValue !== 'boolean') {
+      return;
+    }
+    // @ts-ignore
+    if (this.config?.experimental?.enableTcpReachability !== newValue) {
+      // @ts-ignore
+      this.config.experimental.enableTcpReachability = newValue;
```

Comment:
based on the `if` both `config` and `experimental` can be undefined. it would be better to use [Lodash set](https://lodash.com/docs/4.17.15#set) or similar

Reply:
actually I've removed the optional chaining. There are lots of other existing places in SDK where config properties are accessed without optional chaining and the meetings plugin has a default config defined with all these props.